deploy/kubernetes/manifests/payment-dep.yaml

```diff
@@ -21,7 +21,7 @@ spec:
             cpu: 100m
             memory: 100Mi
           requests:
-            cpu: 101m
+            cpu: 99m
```

Comment:
The 101 was just to test the CI around config. I had meant to revert it.  But why 99?